### PR TITLE
Fix default Display data

### PR DESCRIPTION
**Pull request by bordoni**
_Ref:_ [C#41738](https://central.tri.be/issues/41738)

### CODE REVIEW

File: src/Tribe/Main.php [hunk 1]
```diff
@@ -1610,15 +1610,21 @@ public function addVenueAndOrganizerEditor() {
 		 */
 		public function displayEventVenueDropdown( $post_id ) {
 			$venue_id = get_post_meta( $post_id, '_EventVenueID', true );
+
+			$is_community_edit = false;
+			if ( class_exists( 'Tribe__Events__Community__Main' ) ) {
+				$is_community_edit = Tribe__Events__Community__Main::instance()->isEditPage;
+			}
```
Comment (borkweb) [inline src/Tribe/Main.php]: Since this is 4 lines with the smallest amount of logic, it feels like it should be a method.

Comment (bordoni) [inline src/Tribe/Main.php]: Agreed, will work on dat.

**Comment by bordoni** [comment]
Updated this code.

**Comment by borkweb** [comment]
:bacon:

**Pull request merged**